x-pack/plugins/security_solution/public/resolver/view/panels/node_list.tsx

```diff
@@ -145,10 +99,32 @@ export const ProcessListWithCounts = memo(() => {
         },
       },
     ],
-    [pushToQueryParams, handleBringIntoViewClick, isProcessTerminated]
+    []
   );
 
   const { processNodePositions } = useSelector(selectors.layout);
+  const nodeHrefs: Map<SafeResolverEvent, string | null | undefined> = useSelector(
+    (state: ResolverState) => {
```

Comment:
We should wrap this in `useMemo`, otherwise it's going to run on every render and its `O(n)` or worse.